doc/part-1-github-ui/github-contributions.md

```diff
@@ -8,6 +8,41 @@ description: 'PART 1: GITHUB UI'
 
 At this point, the focus is not about Git and Github best practices. The learning videos here will cover the basic things needed to get started with contributing to any project using the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos.
 
+### You will learn the following in this section:
+
+1. **Forking a repository.**
```

Comment:
#### Change request 3
In my experience, people don't have much patience for reading lengthy content. But because this idea will be useful for something else when changes to the project starts, then do this:
- Remove this comprehensive list that you have made from the github contributions page. 
- Create a new file directly in the doc folder, and put the comprehensive list in there.
- Back to the github contributions page. Let's just make the list simple. Create the list using the captions under the videos. Don't change anything, just use the captions exactly as they are written. Use unordered list. 

That's all. Let me know when you have made the changes @aaryansr26

Reply:
What should I name the new file for the comprehensive list? @Ifycode

Reply:
You can use `syllabus-idea.md` @aaryansr26